packages/vite/src/node/constants.ts

```diff
@@ -104,6 +104,8 @@ export const KNOWN_ASSET_TYPES = [
   // media
   'mp4',
   'webm',
+  'mkv',
+  'm4v',
```

Comment:
`mrmime.lookup` fails for both extension. Please add a mime type to `registerCustomMime` in `packages/vite/src/node/plugin/assets.ts` as it's described in line 88 - 90.